{"status": "broken", "name": "Using a list comprehension in a WITH", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "ListComprehension"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486036250, "historyId": "7a45b4b5d23796797c9783d7512ffdd6", "start": 1578486036123, "steps": [{"status": "passed", "start": 1578486036123, "stop": 1578486036244, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486036245, "stop": 1578486036248, "name": "And having executed", "attachments": [{"source": "74cf088e-3092-497f-9054-57f29c89036f-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486036249, "stop": 1578486036249, "name": "When executing query", "attachments": [{"source": "dec9aa9a-4d53-4dc4-b9fd-3735d3b2942e-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "0c6723eb-acf4-4946-92dd-8a92ad31a5fc-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486036250, "start": 1578486036250, "statusDetails": {"message": "ResponseError: RedisGraph does not currently support list comprehension\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486036250, "stop": 1578486036250, "name": "And no side effects"}], "statusDetails": {"message": "ResponseError: RedisGraph does not currently support list comprehension\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "445312b5-b145-4a51-b112-f5dc6aee95dc"}